samples/python/topology/state/find_outliers_functions.py

```diff
@@ -1,5 +1,5 @@
 import random
-import statistics
+import math
```

Comment:
The `statistics` package has functions to compute the mean and stddev given a list of tuples.  Since we don't want to keep a history of all tuples in memory, we are changing the logic to compute the mean and stddev incrementally.  This requires some functions in the `math` library (like math.sqrt()), and the `statistics` package is no longer needed.
Computing the mean and stddev incrementally is already done by the Java sample.  I didn't do a proper port with the first delivery of the Python sample.